src/node_api.cc

```diff
@@ -1117,112 +1140,125 @@ napi_status napi_get_prototype(napi_env e,
                                           size_t length,
                                           napi_value* result) {
-  NAPI_PREAMBLE(e);
+  NAPI_PREAMBLE(env);
   CHECK_ARG(result);
 
   *result = v8impl::JsValueFromV8LocalValue(
-      v8::Array::New(v8impl::V8IsolateFromJsEnv(e), length));
+      v8::Array::New(v8impl::V8IsolateFromJsEnv(env), length));
 
   return GET_RETURN_STATUS();
 }
 
-napi_status napi_create_string_utf8(napi_env e,
-                                    const char* s,
-                                    size_t length,
+napi_status napi_create_string_utf8(napi_env env,
+                                    const char* buf,
+                                    size_t bufsize,
                                     napi_value* result) {
-  NAPI_PREAMBLE(e);
+  NAPI_PREAMBLE(env);
   CHECK_ARG(result);
 
-  auto isolate = v8impl::V8IsolateFromJsEnv(e);
+  auto isolate = v8impl::V8IsolateFromJsEnv(env);
   v8::Local<v8::String> str;
-  CHECK_NEW_FROM_UTF8_LEN(isolate, str, s, length);
+  CHECK_NEW_FROM_UTF8_LEN(isolate, str, buf, bufsize);
 
   *result = v8impl::JsValueFromV8LocalValue(str);
   return GET_RETURN_STATUS();
 }
 
-napi_status napi_create_string_utf16(napi_env e,
-                                     const char16_t* s,
-                                     size_t length,
+napi_status napi_create_string_utf16(napi_env env,
+                                     const char16_t* buf,
+                                     size_t bufsize,
                                      napi_value* result) {
-  NAPI_PREAMBLE(e);
+  NAPI_PREAMBLE(env);
   CHECK_ARG(result);
 
-  auto isolate = v8impl::V8IsolateFromJsEnv(e);
+  auto isolate = v8impl::V8IsolateFromJsEnv(env);
   auto str_maybe =
       v8::String::NewFromTwoByte(isolate,
-                                 reinterpret_cast<const uint16_t*>(s),
+                                 reinterpret_cast<const uint16_t*>(buf),
                                  v8::NewStringType::kInternalized,
-                                 length);
+                                 bufsize);
   CHECK_MAYBE_EMPTY(str_maybe, napi_generic_failure);
   v8::Local<v8::String> str = str_maybe.ToLocalChecked();
 
   *result = v8impl::JsValueFromV8LocalValue(str);
   return GET_RETURN_STATUS();
 }
 
-napi_status napi_create_number(napi_env e, double v, napi_value* result) {
-  NAPI_PREAMBLE(e);
+napi_status napi_create_number(napi_env env, 
+                               double value, 
+                               napi_value* result) {
+  NAPI_PREAMBLE(env);
   CHECK_ARG(result);
 
   *result = v8impl::JsValueFromV8LocalValue(
-      v8::Number::New(v8impl::V8IsolateFromJsEnv(e), v));
+      v8::Number::New(v8impl::V8IsolateFromJsEnv(env), value));
 
   return GET_RETURN_STATUS();
 }
 
-napi_status napi_create_boolean(napi_env e, bool b, napi_value* result) {
-  NAPI_PREAMBLE(e);
+napi_status napi_get_boolean(napi_env env, bool value, napi_value* result) {
+  NAPI_PREAMBLE(env);
```

Comment:
Fair enough. I had a similar thought while making these changes but figured it would make more sense to bundle all such NAPI_PREAMBLE cleanups together. These couple are really simple, though. Will add them.